### DISCUSSION TITLE
Filter backlinks when in Zoom

### DISCUSSION THREAD

**Discussion by PeterTheobald** [vslinko/obsidian-zoom#4] [Ideas]
It would be great for use in very large outlines if the Backlinks display got filtered when in Zoom mode. The backlinks should show only those backlinks linking to the currently zoomed block (or only visible blocks) instead of all backlinks to the current file (note)

**Comment by vslinko** [comment]
@PeterTheobald Thank you for your contribution!

Unfortunately, It’s hard to implement so I converted your issue to discussion.

**Comment by vslinko** [comment]
Discussion was transferred from https://github.com/vslinko/obsidian-outliner to https://github.com/vslinko/obsidian-zoom
